snapcraft/_store.py

```diff
@@ -495,6 +498,14 @@ def list_keys():
             tablefmt="plain",
         )
         print(tabulated_keys)
+    elif not keys and enabled_keys:
+        registered_keys = "\n".join([f"- {key}" for key in enabled_keys])
+        print(
+            "No keys have been created on this system. "
```

Comment:
Maybe?
```suggestion
            "No keys found. "
```

Interesting that snapd doesn't seem to have an interface to import a key you exported?  Is it just expected you create a new key on a new host rather than maintain them?